app/schedule/clean_tmpdir.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const path = require('path');
+const fs = require('mz/fs');
+const rimraf = require('mz-modules/rimraf');
+const moment = require('moment');
+
+module.exports = app => {
+  return class CleanTmpdir extends (app.Subscription || app.BaseContextClass) {
+    static get schedule() {
+      return {
+        type: 'worker',
+        cron: app.config.multipart.cleanSchedule.cron,
+        immediate: false,
+        // disable on stream mode
+        disable: app.config.multipart.mode === 'stream',
+      };
+    }
+
+    async _remove(dir) {
+      const { ctx } = this;
+      if (await fs.exists(dir)) {
+        ctx.coreLogger.info('[egg-multipart:CleanTmpdir] removing tmpdir: %j', dir);
```

Comment:
how about using `app.loggers.scheduleLogger` ?

https://github.com/eggjs/egg-schedule/blob/master/app.js#L9

just for discuss, I don't have the tendency.

Reply:
No